egs2/iemocap/asr1/local/score.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# Copyright 2021  Siddhant Arora
```

Comment:
If it is the same as other recipes, you could add a softlink instead of copy it.

Reply:
This one is also modified for this recipe. (most of the parts are taken from https://github.com/espnet/espnet/blob/master/egs2/fsc/asr1/local/score.py)